File: DotNet/CesiumLanguageWriter/Advanced/ICesiumUnitCartesian3ListValuePropertyWriter.cs
using System.Collections.Generic;

namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a list of three-dimensional unit magnitude Cartesian values.
    /// </summary>
    public interface ICesiumUnitCartesian3ListValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a list of three-dimensional unit magnitude Cartesian values.
        /// </summary>
        /// <param name="values">The values.</param>
        void WriteUnitCartesian(IEnumerable<UnitCartesian> values);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumUnitCartesian3ListValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;

/**
 * A writer that can write a value as a list of three-dimensional unit magnitude Cartesian values.
 */
@SuppressWarnings({
    "unused",
    "deprecation",
    "serial"
})
public interface ICesiumUnitCartesian3ListValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a list of three-dimensional unit magnitude Cartesian values.
    * @param values The values.
    */
    void writeUnitCartesian(Iterable<UnitCartesian> values);
}